client/my-sites/customer-home/main.jsx

```diff
@@ -248,12 +252,14 @@ class Home extends Component {
 								label={ translate( 'Change theme' ) }
 								iconSrc="/calypso/images/customer-home/theme.svg"
 							/>
-							<ActionBox
-								href={ menusUrl }
-								onClick={ () => trackAction( 'my_site', 'edit_menus' ) }
-								label={ translate( 'Edit menus' ) }
-								iconSrc="/calypso/images/customer-home/menus.svg"
-							/>
+							{ showCustomizer && (
+								<ActionBox
+									href={ menusUrl }
+									onClick={ () => trackAction( 'my_site', 'edit_menus' ) }
+									label={ translate( 'Edit menus' ) }
+									iconSrc="/calypso/images/customer-home/menus.svg"
+								/>
+							) }
```

Comment:
@apeatling what if we dropped this change here? The Navigation block is just a placeholder, so technically folks would need to go to the customizer still for any Menu edits

Reply:
The trouble is, it's going to be very confusing for them clicking this button.

The user will not have any menus set up, so even if they have multiple pages in their navigation, they'll have to recreate it from scratch in the customizer and will see "Empty Menu".

<img width="724" alt="Screen Shot 2019-08-29 at 4 05 59 PM" src="https://user-images.githubusercontent.com/1464705/63982353-0082c580-ca77-11e9-9af7-edfb329543b3.png">
<img width="723" alt="Screen Shot 2019-08-29 at 4 05 53 PM" src="https://user-images.githubusercontent.com/1464705/63982354-0082c580-ca77-11e9-8553-423ec9c52d19.png">